packages/js/plugins/ipfs/package.json

```diff
@@ -22,11 +22,12 @@
   "dependencies": {
     "@web3api/core-js": "0.0.1-prealpha.18",
     "cids": "^1.1.4",
-    "ipfs-http-client-lite": "0.3.0",
+    "@dorgjelli-test/ipfs-http-client-lite": "0.3.1",
```

Comment:
This forked package has the following modification to it: https://github.com/ipfs-shipyard/js-ipfs-http-client-lite/pull/5